ArduPlane/Parameters.h

```diff
@@ -347,6 +347,9 @@ class Parameters {
         k_param_mixing_offset,
         k_param_dspoiler_rud_rate,
 
+        // 250: ArduSoar parameters
+        k_param_soaring_controller,
```

Comment:
Moved soaring_controller to G2, but it would be good if you took a look at the result to make sure it's right.